cli/sync_complete.go

```diff
@@ -0,0 +1,39 @@
+package cli
+
+import (
+	"golang.org/x/xerrors"
+
+	"github.com/coder/coder/v2/agent/agentsocket"
+	"github.com/coder/coder/v2/cli/cliui"
+	"github.com/coder/serpent"
+)
+
+func (*RootCmd) syncComplete() *serpent.Command {
+	return &serpent.Command{
+		Use:   "complete <unit>",
+		Short: "Signal that a unit has completed",
+		Long:  "Mark a unit as complete, indicating it has completed its work. This allows units that depend on it to proceed with their startup.",
```

Comment:
Signal/Mark still causes confusion here, It's fine to repeat the same text in short and long, just making it shorter for the former. I don't like this mixed usage for our users.

Thinking as a user: Does it signal? Does it mark? What does either mean?